vuepress/docs/v7.0/docs/reference/deployment-structure.md

```diff
@@ -4,154 +4,128 @@ sidebarDepth: 2
 
 * GitHub: <https://github.com/entando-k8s/entando-k8s-dbjob>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/entando-k8s-dbjob>
 
-The screenshot below highlights the init containers for the Entando application schema creation, db 
-initialization, and component repository database.
+The screenshot below highlights the init containers for the Entando Application schema creation, DB initialization, and component repository database.
 
 ![Init Containers Screenshot](./img/init-containers-screenshot.png)
 
-Many managed kubernetes instances like OpenShift won’t show init containers in their dashboards. So if you’re 
-troubleshooting you may need to look deeper. When fetching logs for an init container using kubectl, you must 
-pass the container name as an argument to the call. For example,
+Many managed Kubernetes instances like OpenShift don’t display init containers on their dashboards. If you’re troubleshooting, the logs may provide some useful information. To fetch logs for an init container using kubectl, you must 
+pass the container name as an argument. 
 
-        kubectl logs <pod> -c <container> -n <namespace>        
+For example:
+
+        kubectl logs YOUR-POD -c YOUR-CONTAINER-NAME -n YOUR-NAMESPACE        
         kubectl logs default-sso-in-namespace-deployment-db-preparation-job-ddbdbddb-a  -c default-sso-in-namespace-deployment-db-schema-creation-job -n sprint1-rc
 
 #### Customization
-It is unlikely that the init containers will be customized as part of an Entando project. The init containers 
-will automatically restore a backup included in your application so that you can create custom images that 
-include your application setup. 
-See [Backing Up and Restoring Your Environment](../../tutorials/devops/backing-up-and-restoring-your-environment.md).
+The init containers automatically restore a backup included in your application so that you can create custom images that include your application setup. 
+See [Backing Up and Restoring Your Environment](../../tutorials/devops/backing-up-and-restoring-your-environment.md) for more details.
 
 
 ## entando-de-app
-The _entando-de-app_ is a J2EE application and is an instance of the _entando-core_ (see a description of the 
-_entando-core_ repo below). Reviewing the dependencies of this application in the pom.xml will reveal the 
-dependencies on the _entando-core_, _entando-engine_, and _admin-console_ which encompass the core 
-functionality in versions of Entando prior to Entando 6. In a quickstart deployment the _entando-de-app_ is deployed as part of the _entando-composite-app_ multi 
-container pod.
+The **entando-de-app** is a J2EE application and an instance of the [**entando-core**](#entando-core). It provides the server image the Entando Operator requires to manage the deployment and provide pathways for other Entando Components. The pom.xml for the application reveals its dependencies.
```

Comment:
"It provides the Entando Operator with the server image to manage the deployment and create pathways for other Entando components."

"...reveals entando-de-app dependencies" -> otherwise reads the file reveals the file's dependencies